docs/quantum_chess/concepts.ipynb

```diff
@@ -0,0 +1,2063 @@
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TW7je1uhcLw2"
+   },
+   "source": [
+    "This colab illustrates the difficulties of running on NISQ hardware using the example of Quantum Chess.  \n",
+    "\n",
+    "This tutorial assumes basic knowledge of the following:\n",
+    "\n",
+    "*   Basic quantum computing principles, specifically, what unitary matrices are, and what an $\\text{iSWAP}$ (and $\\sqrt{\\text{iSWAP}}$ gate is)\n",
+    "*   The basic movement rules of [Chess](https://en.wikipedia.org/wiki/Chess#Movement) and the [Algebraic notation](https://en.wikipedia.org/wiki/Algebraic_notation_(chess)) method of naming chess squares and moves.\n",
+    "\n",
+    "\n",
+    "![Algebraic notation](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b6/SCD_algebraic_notation.svg/242px-SCD_algebraic_notation.svg.png)\n",
+    "\n",
+    "*   The [Cirq](https://github.com/quantumlib/Cirq) framework for constructing quantum circuits.\n",
+    "*   While the tutorial will briefly cover the rules of quantum chess, those who are unfamiliar with this game may [try it out](https://quantumchess.net/play/) or read the exact rules in the [Quantum Chess paper](https://arxiv.org/abs/1906.05836).  You can also find a sample implementation in this repository under recirq.quantum_chess.\n",
+    "\n",
+    "The goal of this tutorial is to illustrate some basic ideas of transforming a theoretical algorithm (such as Quantum Chess) into a real hardware algorithm, without getting lost in details of more complicated domain-specific problems.  \n",
+    "\n",
+    "The tutorial is split into several parts:\n",
+    "\n",
+    "1.   **Introduction to Quantum Chess**: how quantum chess maps to qubits and quantum operations\n",
+    "2.   **Quantum chess in Cirq**: how to execute these operations using the cirq framework\n",
+    "3.   **Handling partial measurement**:  how to handle moves that include measurement that partially collapses the board state\n",
+    "4.   **Noise**: how to deal with some simple noise models\n",
+    "5.   **Decomposition**: how to decompose gates\n",
+    "6.   **Qubit Layout**: how to run circuits when connectivity matters.\n",
+    "\n",
+    "You will learn about the concepts of **circuit construction**, **post-selection**, **gate decomposition**, **noise**, **optimizers**, and **qubit mappings**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "49QJ1Lx1en5z"
+   },
+   "source": [
+    "First, let's get started by installing cirq on our colab runtime:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "ugH2oPsQM0Po"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -U cirq"
```

Comment:
I think this should be recirq. Also please change to

```python
try:
    import recirq
except ImportError:
    !pip install git+https://github.com/quantumlib/ReCirq
```

so we don't clobber users' environment (read: my environment) if and when people check these out and run locally

(opened https://github.com/quantumlib/ReCirq/issues/83 to track this change in the other notebooks)

Although nothing in this notebook uses recirq functionality, see review summary